static/src/javascripts/projects/common/modules/commercial/creatives/expandable-v3.js

```diff
@@ -106,11 +106,14 @@ define([
     ExpandableV3.prototype.listener = function () {
         var that = this;
         if (!this.initialExpandCounter && (window.pageYOffset + bonzo.viewport().height) > (this.$adSlot.offset().top + this.openedHeight)) {
-            if (!storage.local.get('gu.commercial.expandable.' + this.params.ecid)) {
+            var itemId = $('.ad-slot__content', $(that.$adSlot)).attr('id'),
```

Comment:
Isn't a `that.$adSlot` already a bonzo object? If not, rename it to `that.$adSlot`. Otherwise, you don't need to wrap it in `$()`.